include/st_api.h

```diff
@@ -18,6 +18,14 @@
 extern "C" {
 #endif
 
+#ifdef __GNUC__
+#define PACK(__Declaration__) __Declaration__ __attribute__((__packed__))
```

Comment:
can you move PACK to mt_API.h and add one prefix, ex, MTL_PACK

Reply:
ok will do

Reply:
done